ecosystem/resources/members/qiskit-optimization.toml

```diff
@@ -1,6 +1,6 @@
 name = "Qiskit Optimization"
 url = "https://github.com/qiskit-community/qiskit-optimization"
-description = "Framework that covers the whole range from high-level modeling of optimization problems, with automatic conversion of problems to different required representations, to a suite of easy-to-use quantum optimization algorithms that are ready to run on classical simulators, as well as on real quantum devices via Qiskit."
+description = "Model optimization problems, convert them between different representations, and solve them by using quantum optimization algorithms on simulators or systems."
```

Comment:
Same thought about simulators here since they will be sunset -- just remove the "simulators or"?